dockets/lua/move_delayed_items.lua

```diff
@@ -0,0 +1,26 @@
+local delayedQueueKey = KEYS[1]
+local payloadKey = KEYS[2]
+local queueKey = KEYS[3]
+
+-- We're forced to pass this in because Redis forces our
+-- scripts to be deterministic
+local currentTime = ARGV[1]
+local FIFO = ARGV[2]
+
+local toMove = redis.call('zrangebyscore', delayedQueueKey, 0, currentTime)
+if #toMove == 0 then
+   return false
```

Comment:
Is it actually worth it to return a value from this script? It might be better to kill this block and the `return` at the end.